Silicon/ElkhartlakePkg/ElkhartlakePkg.dec

```diff
@@ -18,4 +18,7 @@
 [Guids]
   gSiMemoryPlatformDataGuid                = { 0x6210d62f, 0x418d, 0x4999, { 0xa2, 0x45, 0x22, 0x10, 0x0a, 0x5d, 0xea, 0x44 } }
   gSchemaListGuid                          = { 0x3047c2ac, 0x5e8e, 0x4c55, {0xa1, 0xcb, 0xea, 0xad, 0x0a, 0x88, 0x86, 0x1b } }
+  gPlatformElkhartLakeTokenSpaceGuid       = { 0xdce79bb5, 0x9f3f, 0x45f2, {0xa8, 0xac, 0x36, 0xf2, 0x27, 0x06, 0x04, 0x9a } }
 
+[PcdsFixedAtBuild]
+  gPlatformElkhartLakeTokenSpaceGuid.PcdIsReleaseBuild    | FALSE | BOOLEAN | 0x30000100

```

Comment:
Consider using DebugCodeEnabled() / DEBUG_CODE() instead of adding new PCD.

Reply:
Ok noted will use DebugCodeEnabled() instead